test/box-luatest/gh-11224_fix_commit_wait_none_journal_queue_test.lua

```diff
@@ -0,0 +1,60 @@
+local server = require('luatest.server')
```

Comment:
Nit: we usually name tests starting with `gh_`, not `gh-`.

Reply:
Fixed.